packages/spark-core/components/_buttons.scss

```diff
@@ -60,9 +61,8 @@
 .sprk-c-Button--tertiary {
   background-color: $btn-tertiary-background-color;
   box-shadow: none;
-  border-color: $btn-tertiary-border-color;
+  border: none;
```

Comment:
I think the variable should be left here but zeroed out in the _settings.scss file. You can create a new name to reflect the whole border property.